### ISSUE TITLE
About the metrics

### ISSUE THREAD

**Issue by zhuhrun** [dvlab-research/Seg-Zero#99]
Hello, excellent work.

I seem to be encountering some issues. I trained the model on an A800-80GB GPU. 
During my training, the metric `“critic/rewards/mean”` appears difficult to exceed `5`. I'm curious to know what values the metrics `“critic/rewards/mean”` and `“critic/rewards/min”` typically reach in your training?May i have your training log.

```python
# reward
"critic/rewards/mean": torch.mean(sequence_reward).detach().item(),
"critic/rewards/max": torch.max(sequence_reward).detach().item(),
"critic/rewards/min": torch.min(sequence_reward).detach().item(),
``` 

Thank you for your prompt reply.

**Comment by LiuRicky** [comment]
These metrics are approximately shown as the following.

<img width="2076" height="374" alt="Image" src="https://github.com/user-attachments/assets/a649d88f-e056-47c0-8bf9-b131f64d7028" />

**Comment by zhuhrun** [comment]
> These metrics are approximately shown as the following.
> 
> <img alt="Image" width="2000" height="374" src="https://private-user-images.githubusercontent.com/44423567/486151819-a649d88f-e056-47c0-8bf9-b131f64d7028.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTcxMjcyNTIsIm5iZiI6MTc1NzEyNjk1MiwicGF0aCI6Ii80NDQyMzU2Ny80ODYxNTE4MTktYTY0OWQ4OGYtZTA1Ni00N2MwLThiZjktYjEzMWY2NGQ3MDI4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA5MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwOTA2VDAyNDkxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAzNGUxYmY1ZTc5MjNlMmJlNzUwNzY2ZGEzZTA5YzJmN2QyMjY1OTkzMDYzMjk3NjZlZWZkNDhiZDU5OGNmZDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.n4EIt7Vepov69TGzdiW2v2FAm_xJBw8Vz2l7vf5Zryc">

Thank you for your reply.

Due to hardware limitations, I employed fine-tuning with `Lora` parameters rather than full parameter fine-tuning within the `VERL` framework. As in `VERL`,` Lora` does not support the visual `VIT` component, meaning these parameters are frozen. I think this might be the reason why metric improvements are difficult to achieve.
I'm curious if you've tried fine-tuning LORA? Also, I'd like to know how much time the training took in total for your experiment?

Thank you for your prompt reply again.
